### PR TITLE
(PXP-7718): feat/hideEmptyFilterSlider: added logic to remove empty filters

### PR DESCRIPTION
**Pull request by ocshawn**
Jira Ticket: [PXP-7718](https://ctds-planx.atlassian.net/browse/PXP-7718)

### Improvements
If slider is blank, do not display. Checking for `"count": 0`

### CODE REVIEW

File: src/components/Utils/filters.js [hunk 1]
```diff
@@ -75,15 +75,24 @@ export const updateCountsInInitialTabsOptions = (
           if (flattenProcessedTabsOptions[`${field}`]
             && flattenProcessedTabsOptions[`${field}`].length > 0
             && updatedTabsOptions[`${actualFieldName}`].histogram) {
-            updatedTabsOptions[`${actualFieldName}`].histogram[0].count = flattenProcessedTabsOptions[`${field}`][0].count;
+            const currentFlattenProcessedTabsOptions = flattenProcessedTabsOptions[`${field}`][0];
+
+            // if empty range histogram should be removed so filter is not shown
+            if (currentFlattenProcessedTabsOptions.key[0] === null
+              && currentFlattenProcessedTabsOptions.key[1] === null
+              && currentFlattenProcessedTabsOptions.count === 0) {
```
Comment (mfshao) [inline src/components/Utils/filters.js:83]: should be enough if we just check for `count === 0` ?

Comment (ocshawn) [reply src/components/Utils/filters.js:83]: I was being extra cautious, will run some tests and see if anything ever pops up with a count that does not have a range

Comment (ocshawn) [reply src/components/Utils/filters.js]: updated to only check count

**Review by mfshao — APPROVED** [review]
lgtm 👍

**Pull request merged**